.github/workflows/ci.yml

```diff
@@ -0,0 +1,117 @@
+name: Continuous Integration testing
+on:
+  push:
+    branches:
+      - meson
```

Comment:
This should probably be changed to `master` so that CI runs on pushes to `master` and not the `meson` branch